### ISSUE TITLE
Correct option differences between Network/Differential and Unifying roads conflation

### ISSUE THREAD

**Issue by bwitham** [ngageoint/hootenanny-ui#969]
**Don't start on this until https://github.com/ngageoint/hootenanny/pull/2148 is merged.**

- When the Unifying roads engine is selected, rubber sheeting should be turned off (leave it on for Network/Diff, which is default)
- When the Unifying roads engine is selected, the CornerSplitter cleaning op should be disabled (leave it on for Network/Diff, which is default)
- Both the rubber sheeting minimum ties minvalues under "Waterway Options" and "Rubber waterway rubber sheeting minimum ties minvalue should be = 1, not zero.

When this and the following options are done, the autocorrect.options parameter can be removed from both the UI and the core:
* #970
* #971 
* #972

**Comment by bwitham** [comment]
Put this temporarily on hold.  The outcome of https://github.com/ngageoint/hootenanny/issues/2190 may affect it.  Should know by today or tomorrow what to do with this task.
